src/RouterFactory.php

```diff
@@ -2,14 +2,26 @@
 
 namespace Yiisoft\Router;
 
+use InvalidArgumentException;
 use Psr\Container\ContainerInterface;
 
+use function count;
+use function is_array;
+use function is_callable;
+use function is_string;
+
 final class RouterFactory
 {
+    /**
+     * @var callable
```

Comment:
How about typing the property?

Reply:
Properties don't support `callable` type for now

Reply:
What about `\Closure`?